sdk/python/jobs/single-step/pytorch/train-hyperparameter-tune-deploy-with-pytorch/train-hyperparameter-tune-deploy-with-pytorch.ipynb

```diff
@@ -1,718 +1,718 @@
 {
```

Comment:
is it possible to change indent back to 2?

Reply:
seems black check will format it like this. If change indent back to 2, smoke CI will fail